GeorgianMemenator/Scenes/DogBreedsList/DogBreedsListView.swift

```diff
@@ -21,14 +20,19 @@ struct DogBreedsListView: View {
             } else if viewModel.isLoading {
                 ActivityIndicator(isAnimating: true)
             } else {
-                List(viewModel.dogBreeds) { dogBreed in
-                    VStack(alignment: .leading, spacing: 4) {
-                        Text(dogBreed.name)
-                        Text("min life: \(dogBreed.minLife), max life: \(dogBreed.maxLife)")
+                NavigationView {
```

Comment:
You got the idea, we definitely need to have navigations, but I want to explain you something.

1. `NavigationView` is a view which helps to chain views and allow to navigate back to previous view. But we need only one `NavigationView` in one flow. So this is the hierarchy that we have now:

Whenever user taps on dog breeds list in `MainView`, we show `NavigationView` with the `DogBreedsListView`, whenever user touches item in `DogBreedsListView` we can ask `NavigationView` to show another view.

So, since you added `NavigationView` here this is what you get:

<img width="200" alt="image" src="https://user-images.githubusercontent.com/9494262/227765656-ce44499a-4fee-4e9c-a350-e7a1e9ee4f89.png">

But, since we already have created one `NavigationView` in `MainView` we can delete `NavigationView` have just `NavigationLink`. At the end you will have view hierarchy like that:

<img width="200" alt="image" src="https://user-images.githubusercontent.com/9494262/227765745-84f66d15-5db8-4134-92bc-8a782ce524d9.png">

If you look closely you will see that there two back buttons, which is not ideal.

Reply:
@qetiavanashvili please read it carefully and let me know if you need more information.

Reply:
After some mimutes, I wrote this . I hope it will be correct
![Uploading Screenshot 2023-03-27 at 14.49.15.png…]()

Reply:
`DogBreedDetailView` requires to pass viewmodel in this initaliser, thats why it is complaining:

Try this: `DogBreedDetailView(viewModel: DogBreedDetailViewModel())`